index.php

```diff
@@ -11,6 +11,9 @@
     $ucastnici[] = $novyUcastnik;
 
     file_put_contents("ucastnici.txt", implode("\n", $ucastnici));
+
+    header("Location: /");
```

Comment:
🤔
akt nevim, jestli to resit teda. Jakoze kazdej F5 se jim tam nekdo prida, to je fakt, ale tohle nemame sanci vysvetlit co to dela :upside_down_face:

Reply:
mam to ja a snad to nejak zvladnu..